frontend/tests/controllers/CoderOfTheMonthTest.php

```diff
@@ -570,4 +573,590 @@ public function testMentorCanChooseCoderOfTheMonth() {
         );
         $this->assertFalse($canChooseCoder);
     }
+
+    private function updateIdentity($identity, $gender) {
```

Comment:
puedes ponerle tipos?

```suggestion
    private function updateIdentity($identity: \OmegaUp\DAO\VO\Identities, $gender: string): void {
```

Reply:
listo